react-forms/my-app/src/RegistrationFormUncontrolled.tsx

```diff
@@ -0,0 +1,18 @@
+import { FormEvent } from 'react';
+
+export function RegistrationFormUncontrolled() {
+  function handleSubmit(event: FormEvent) {
+    event.preventDefault();
+    const formData = new FormData(event.currentTarget as HTMLFormElement);
```

Comment:
The best way to type this is to add generics to the type on the `event` parameter:
`handleSubmit(event: FormEvent<HTMLFormElement>)`
then TypeScript will know that `event.currentTarget is an HTMLFormElement and you won't need to use `as`.